imap_processing/cdf/global_attrs.py

```diff
@@ -88,12 +130,21 @@ class GlobalInstrumentAttrs:
         Descriptor of the instrument (Ex: "IDEX>Interstellar Dust Experiment")
     text: str
         Explanation of the instrument, usually as a paragraph.
+    instrument_type: str default="Particles (space)"
```

Comment:
There should be a space before the colon in numpydoc for all the parameters in the docstrings https://numpydoc.readthedocs.io/en/latest/format.html#parameters
```suggestion
    instrument_type : str, default="Particles (space)"
```